<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result Grab </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="s_details">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
font: 75 10pt &quot;Arial&quot;;
 font-weight: bold;
text-align: left;
</string>
     </property>
     <property name="text">
      <string>Advanced Level</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/ravindra_sir/l.png</normaloff>D:/ravindra_sir/l.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="s_enq">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
font: 75 10pt &quot;Arial&quot;;
 font-weight: bold;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Ordinary Level</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/ravindra_sir/l.png</normaloff>D:/ravindra_sir/l.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="s_enq_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
font: 75 10pt &quot;Arial&quot;;
 font-weight: bold;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/ravindra_sir/l.png</normaloff>D:/ravindra_sir/l.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="g">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>551</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>551</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Developed by Shanaka Anuradha | fb.com/shanaka1995 | 0715654548</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
